tests/test_interface.rb

```diff
@@ -464,8 +475,8 @@ def test_interface_speed_change
       interface.speed = 1000
       assert_equal('1000', interface.speed)
       interface_ethernet_default(interfaces_id[0])
-    rescue RuntimeError => e
-      assert_match(/port doesn t support this speed/, e.message)
+    rescue => e
+      speed_change_disallowed(e.message)
```

Comment:
It's not - 1) see prev comment on RuntimeError as default (wrong) ; 2) I need to add a fail/raise following the disallow check.